recipes/mapgl/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mapGL" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f053573cba3f5482620d68464f6fd6d5505e0e82bbcffd8e65d43815829c86d6"
+
+build:
+  number: 0
+  entry_points:
+    - mapGL.py = map_GL.mapGL:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: generic
+
+requirements:
+  host:
+    - pathlib
+    - bx-python
+    - numpy
+    - pip
+    - python >=3
+    - six
+    - cython
+  run:
+    - bx-python
+    - numpy
+    - python >=3
+    - six
+
+test:
+  imports:
+    - map_GL
+  commands:
+    - mapGL.py --help
+
+about:
+  home: "https://github.com/adadiehl/mapGL"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: ""
+  summary: "Prediction of lineage-specific gain and loss of sequence elements using phylogenetic maximum parsimony."
+  doc_url: ""
+  dev_url: ""
```

Comment:
Can you remove the empty doc_url and dev_url entries? If there's no license that comes with the code then remove the license_file entry too.

Reply:
Fixed.